lib/components/AuthorizedForComponent.react.js

```diff
@@ -0,0 +1,28 @@
+import RoleAwareComponent from './RoleAwareComponent.react';
+import React, { PropTypes } from 'react';
+
+export default class AuthorizedFor extends RoleAwareComponent {
+    static propTypes = {
+        roles: PropTypes.array.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.allowedRoles = this.props.roles;
+        // roles of the user - usually fetched during authentication
+        this.userRoles = undefined;
```

Comment:
userRoles are `undefined` - this won't work... `this.rolesMatched()` will always throw exception...